src/ui/App.swift

```diff
@@ -201,9 +203,14 @@ class App: AppCenterApplication, NSApplicationDelegate {
     }
 
     func focusSelectedWindow(_ selectedWindow: Window?) {
-        hideUi()
-        guard let window = selectedWindow, !CGWindow.isMissionControlActive() else { return }
-        window.focus()
+        hideUi(true)
+        guard let window = selectedWindow, !CGWindow.isMissionControlActive() else {
+            previewPanel.orderOut(nil)
+            return
+        }
+        window.focus() {
+            Windows.previewFocusedWindowIfNeeded()
+        }
```

Comment:
This parameter is only passed here, right? Perhaps the method could host it, if it has only 1 possible value, don't you think?